src/Services/FormatTranslatableStringService.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Brackets\AdminGenerator\Services;
+
+class FormatTranslatableStringService
+{
+    /**
+     * Format string as value and remove 'id' if it's in the end.
+     *
+     * @param $string
+     * @return string
+     */
+    public function valueWithoutId($string)
+    {
```

Comment:
But I would do it much easier:
```
        if (ends_with($string, "_id")) {
            return substr($string, 0, -3);
        } else {
            return $string;
        }
```
and I would pass it to the Name formatter trait:
https://github.com/BRACKETS-by-TRIAD/admin-generator/blob/master/src/Generate/Traits/Names.php

where I would use `Str` facade, so I don't need to do all the manipulations like in your code (explode, ucfirst, preg_replace, ...). What do you say?

Reply:
I agree with you. Please, check my last commit. What do you say?